src/ng2-datetime-picker.component.ts

```diff
@@ -408,6 +453,7 @@ export class Ng2DatetimePickerComponent {
   }
 
   public close() {
+    console.log('töttöröö');
```

Comment:
Whoops, sorry! That was from when I was trying to find which event was closing the datepicker. :)